compiler/ballerina-lang/src/main/java/io/ballerina/projects/DocumentContext.java

```diff
@@ -59,15 +59,29 @@ class DocumentContext {
     private final DocumentId documentId;
     private final String name;
     private final String content;
+    private boolean enableSyntaxTree = true;
```

Comment:
The name is conflicting with the input. Let's rename this to `disableSyntaxTree`